packages/components/addon/components/hds/form/sensitive-input/field.hbs

```diff
@@ -0,0 +1,28 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<Hds::Form::Field
+  @layout="vertical"
+  @extraAriaDescribedBy={{@extraAriaDescribedBy}}
+  @isRequired={{@isRequired}}
+  @isOptional={{@isOptional}}
+  @id={{@id}}
+  as |F|
+>
+  {{! Notice: the order of the elements is not relevant here, because is controlled at "Hds::Form::Field" component level }}
+  {{yield (hash Label=F.Label isRequired=F.isRequired isOptional=F.isOptional)}}
+  {{yield (hash HelperText=F.HelperText Error=F.Error)}}
+  <F.Control>
+    <Hds::Form::SensitiveInput::Base
+      @isMultiline={{@isMultiline}}
+      @value={{@value}}
+      @isInvalid={{@isInvalid}}
+      @width={{@width}}
```

Comment:
the `@height` should be forwarded too

Reply:
Great spot! Fixed in [6361cd8](https://github.com/hashicorp/design-system/pull/1468/commits/6361cd83e7bf6fd349a1df8a1fbce220319a109a)